### PR TITLE
docs: move contributing to docs/contribute

### PR DESCRIPTION
**Pull request by jooola**

### CODE REVIEW

File: docs/contributor-manual/design/database.md [hunk 1]
```diff
@@ -49,3 +55,13 @@ stateDiagram-v2
...
+
+## Modifying the database schema
+
+To modify the database schema use the following steps:
+
+1. Modify `legacy/build/schema.xml`.
+2. Run `make -C legacy propel-gen` to regenerate the legacy files.
+3. Run `pre-commit run --all` and `make -C legacy format` multiple times to clean and format the previously generated files.
+4. Update the `api` models to reflect the changes made to the schema.
+5. Create a new schema migration file in the `api/legacy/migrations` directory.
```
Comment (jooola) [inline docs/contributor-manual/design/database.md:67]: This is from the previous CONTRIBUTING.md

File: docs/contribute.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+## Contribute with code
+
+Are you familiar with coding in PHP or Python? Have you made projects in [Liquidsoap](https://www.liquidsoap.info/) and some of the other services we use? Help us improve LibreTime by picking an issue in the [list of bugs and feature requests](https://github.com/libretime/libretime/issues).
+
+Then [fork our repo](https://docs.github.com/en/get-started/quickstart/contributing-to-projects) and [setup a development environment](https://libretime.org/docs/contributor-manual/development-environment.md) to get you started! Make sure to setup [pre-commit](https://libretime.org/docs/contributor-manual/development-workflows.md#pre-commit) to enforce the project best practices before committing new code.
+
+Knowledge on how to use [Github](https://guides.github.com/activities/hello-world/)
+and [Git](https://git-scm.com/docs/gittutorial) will suit you well, use the
+links for a quick 101.
```
Comment (jooola) [inline docs/contribute.md:45]: I removed some of the instruction here (e.g. pre-commit) and moved it to the contributor manual.

File: docs/contributor-manual/development-workflows.md [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+- [Create a fork of the project](https://docs.github.com/en/get-started/quickstart/fork-a-repo).
+- Check out the `main` branch. If you're making a minor or small documentation change you can check out the `stable` branch.
+- Create a new branch based on the checked out branch.
+- Work on your changes locally. Try to keep each commit small to make reviews easier.
+- Lint and test the codebase, for example using the `make lint` or `make test` commands inside the app folder you want to check.
+- Push your branch.
+- Create a pull request in Github.
+- We'll review your request and provide feed back.
+
```
Comment (jooola) [inline docs/contributor-manual/development-workflows.md:17]: This is new, I though it was obvious but having it written somewhere makes it probably easier on newcomers.

File: docs/contributor-manual/development-workflows.md [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+:::info
+
+You can also run pre-commit anytime on all the files using:
+
+```bash
+pre-commit run --all-files
+```
+
+:::
```
Comment (jooola) [inline docs/contributor-manual/development-workflows.md:77]: This is from the CONTRIBUTING.md file

**Review by jooola — COMMENTED** [review]
I've made a few comment to say what changed, hope this helps.

File: docs/README.md [hunk 5]
```diff
@@ -21,3 +21,9 @@ Having trouble? We’d like to help!
 
 - [:question: Try the Forum – it’s got answers to many common questions](https://discourse.libretime.org/).
 - [:bug: Report bugs with LibreTime in our ticket tracker.](https://github.com/libretime/libretime/issues)
+
+## History
+
+LibreTime is a fork of AirTime due to stalled development of the open source version. For background on this, see this [open letter to the Airtime community](https://gist.github.com/hairmare/8c03b69c9accc90cfe31fd7e77c3b07d).
+
+Airtime itself was based on the software called Campcaster. A [document regroup some information](https://github.com/libretime/libretime/blob/main/LEGACY.md) about the early days of Campcaster and Airtime.
```
[the remark this answers is not in the document]
Comment (jooola) [reply docs/README.md:29]: I meant that we have the LEGACY.md file that regroup most of the content related to Campcatser and Airtime before the fork. I probably didn't expressed it properly.

Maybe:
```
Airtime itself was based on the software called Campcaster. You can find more information about the [early days of Campcaster and Airtime in the repository](https://github.com/libretime/libretime/blob/main/LEGACY.md).
```

File: docs/contributor-manual/design/database.md [hunk 6]
```diff
@@ -1,4 +1,10 @@
-# Database schema creation and migrations
+---
+title: Database
+---
+
+LibreTime is designed to work with a [PostgreSQL](https://www.postgresql.org/) database server and uses both the [PropelORM](https://github.com/propelorm/Propel) in the legacy app and [Django ORM](https://docs.djangoproject.com/en/4.1/topics/db/models/) in the API to interact with the database.
+
+## Database schema creation and migrations
 
 The method to maintain the database schema, is to write both a migration file for already installed databases and to update a `schema.sql` file for fresh databases. On fresh installation, the database is filled with the `schema.sql` and `data.sql` files and LibreTime won't run any sql migration on top of it. Previously, when LibreTime was upgraded, the missing migrations were run using a custom php based migration tool, those migrations are now handled by Django. The missing migrations are tracked using both a `schema_version` field in the `cc_pref` table and a Django migration id.
```
[the remark this answers is not in the document]
Comment (jooola) [reply docs/contributor-manual/design/database.md:9]: We still maintain `schema.sql` file using propel, and write migrations using django and bump the `schema_version`.

Fresh databases only run the initial migration (load `schema.sql` and set `schema_version` to the latest version).
During upgrades, we use the django migrations and bump `schema_version` on each migration, until we reach the latest `schema_version`.

So it's still actual.
I thought we should only stop maintaining the `schema_version` and using raw sql migration after we set `managed = True` on the django models. But the models doesn't exactly reflect the `schema.sql` yet.

File: docs/README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (paddatrapper) [reply docs/README.md]: Ah I see. Yes, that is probably better wording

**Comment by paddatrapper** [comment]
Sorry, fell off my radar

**Pull request merged**